includes/wizards/class-wizard.php

```diff
@@ -70,7 +77,7 @@ abstract class Wizard {
 	 * $my_wizard = new My_Wizard( [ 'sections' => [ 'my-wizard-section' => 'Newspack\Wizards\My_Wizard\My_Wizard_Section' ] ] );
 	 */
 	public function __construct( $args = [] ) {
-		add_action( 'admin_menu', [ $this, 'add_page' ] );
+		add_action( 'admin_menu', [ $this, 'add_page' ], $this->admin_menu_hook_priority );
```

Comment:
If the hook order is supposed to follow the same logic as the `$menu_order`, we could avoid an additional variable and go with:

```suggestion
		add_action( 'admin_menu', [ $this, 'add_page' ], 2 + $this->menu_order );
```

wdyt?

Reply:
I think we need to keep them separate.  The `$menu_order` has to do with the `custom_order` of wp-admin display menu order.  The values will be in the range of `0 through 5`.

Whereas `$admin_menu_hook_priority` has to do with when `add_page` function is run.  The values here would be above or below the default action hook priority of `10`.

Reply:
The first wizard has a hook priority of 2, which is why I'm assuming that higher than 2 is sufficient and respects the desired execution logic  for following wizards.

I'm still hesitant to add boilerplate code that might not be necessary. Let's defer to @jaredrethman 😀